ci_test/downgrade_to_5.yaml

```diff
@@ -17,7 +17,7 @@
         env: dev
       trace.concentrator:
         extra_aggregators: version
-    datadog_checks:
+    agent_datadog_checks:
```

Comment:
I think this variable name should not change, since this is a public-facing variable

Reply:
In [there](https://github.com/DataDog/ansible-datadog/pull/490/files#diff-62f3d41880f6ed7a6e626fa7224fb9bcf32911acccd8d558cd4297a8624c9c96) it's defined so it's normal